northstar/src/runtime/console.rs

```diff
@@ -160,7 +165,7 @@ impl Console {
         }
 
         // Get a framed stream and sink interface.
-        let mut network_stream = api::codec::Framed::with_capacity(stream, BUFFER_SIZE);
+        let mut network_stream = api::codec::Framed::new_with_max_length(stream, MAX_LINE_LENGTH);
```

Comment:
`api::codec::Framed` was created with some `BUFFER_SIZE` capacity and now is created with a Codec with a max line length. Are those two related? is this still equivalent?

Reply:
Those are not related. The buffer size is just the initial allocation for the buffer while the line length is a hard limit for the codec to wait for a newline terminator. The buffer will grow until max the line length limit.